tasks/sync_repo_languages.py

```diff
@@ -65,9 +66,15 @@ async def run_async(
         except TorngitError:
             log.warning(
                 "Unable to find languages for this repository",
-                dict(repository_id=repository.repoid),
+                extra=dict(repository_id=repository.repoid),
```

Comment:
[nit] i would have the key in the extra dict be repoid here as well

Reply:
Can do!